cg/services/deliver_files/delivery_file_fetcher_service/raw_data_delivery_file_fetcher.py

```diff
@@ -19,7 +22,7 @@
 from housekeeper.store.models import File
 
 
-class FastqDeliveryFileFetcher(FetchDeliveryFilesService):
+class RawDataDeliveryFileFetcher(FetchDeliveryFilesService):
```

Comment:
There are a lot of instances of "fastq" in both the methods, variables and docstrings in this class. Pleas refactor them

Reply:
think I got them all